test/cli/cli.test.js

```diff
@@ -11,7 +11,7 @@ describe('CLI', () => {
     testBin('--progress')
       .then((output) => {
         expect(output.exitCode).toEqual(0);
-        expect(output.stderr).toContain('0% compiling');
+        expect(output.stderr).toContain('100%');
```

Comment:
This change is for webpack@5.

https://dev.azure.com/webpack/webpack-dev-server/_build/results?buildId=9700&view=logs&j=34838ad4-6e10-570d-19ad-fabd2c8dff4b&t=e0d4cf5c-23bc-5ff5-8c16-7ee6929a9592

Reply:
/cc @sokra